build/mark-new-version.sh

```diff
@@ -92,6 +92,10 @@ fi
 
 echo "+++ Running ./versionize-docs"
 ${KUBE_ROOT}/build/versionize-docs.sh ${NEW_VERSION}
+
+echo "+++ Updating swagger"
+${KUBE_ROOT}/hack/update-generated-swagger-docs.sh
```

Comment:
@nikhiljindal done

@caesarxuchao that's `update-generated-docs.sh` not `update-generated-swagger-docs.sh`